src/main/resources/assets/wearablebackpacks/lang/ru_RU.yml

```diff
@@ -0,0 +1,34 @@
+container.wearablebackpacks.backpack=Рюкзак
+
+item.wearablebackpacks.backpack.name=Рюкзак
+
+tooltip.wearablebackpacks.moreInfo=Удерживайте %1$s для информации
+tooltip.wearablebackpacks.openHint=Нажмите %1$s чтобы открыть
+tooltip.wearablebackpacks.equipHint=Поставьте, присядьте и сломайте чтобы надеть.
+tooltip.wearablebackpacks.equipHint.extended=Присядьте и нажмите\nчтобы надеть.
+tooltip.wearablebackpacks.unequipHint=Присядьте и нажмите\nчтобы поставить.
+tooltip.wearablebackpacks.used=Другой игрок уже использует рюкзак
+
+chat.wearablebackpacks.cantEquip=У вас уже надет рюкзак
+chat.wearablebackpacks.cantEquip.chestplate=Вы не можете надеть рюкзак, у вас надета броня
+
+key.wearablebackpacks.open=Открыть рюкзак
+
+
+config.wearablebackpacks.general.equipAsChestArmor=Разрешить надевать как нагрудник
+config.wearablebackpacks.general.equipAsChestArmor.tooltip=Разрешить помещать рюкзак в слот для брони.
+config.wearablebackpacks.general.enableEquippedInteraction=Разрешить взаимодействие
+config.wearablebackpacks.general.enableEquippedInteraction.tooltip=Разрешить открывать рюкзак другим игрокам (и мобам)
+config.wearablebackpacks.general.enableSelfInteraction=Включить самовоздействия
+config.wearablebackpacks.general.enableSelfInteraction.tooltip=Разрешить открывать рюкзак другим игрокам по нажатию.
+config.wearablebackpacks.general.dropAsBlockOnDeath=Выкидывать как блок при смерти
+config.wearablebackpacks.general.dropAsBlockOnDeath.tooltip=Ставить рюкзак как блок, или выкидывать как блок
+
+config.wearablebackpacks.category.backpack=Настройки
+config.wearablebackpacks.category.backpack.tooltip=Настройки рюкзаков
+config.wearablebackpacks.backpack.enabled=ВКЛ
+config.wearablebackpacks.backpack.enabled.tooltip=Включить/Выключить рюкзаки
+config.wearablebackpacks.backpack.durability=Пригодность
+config.wearablebackpacks.backpack.durability.tooltip=Пригодность рюкзаков (0 для бескончености). Если меньше (чем 0), то могут быть проблемы.
```

Comment:
Has some missing info compared to original, mainly that it's only damaged backpacks that might have problems, instead of just vague "might cause problems". Also wording suggests that "setting it to less than zero" is what's causing problems. Might confuse people.